static/app/components/feedback/feedbackItem/messageSection.tsx

```diff
@@ -33,7 +36,14 @@ export default function MessageSection({eventData, feedbackItem}: Props) {
         />
       </Flex>
       <Blockquote>
-        <pre>{feedbackItem.metadata.message}</pre>
```

Comment:
We should keep the `<pre>`. It helps make multi-line messages easier to read:

<img width="764" alt="SCR-20240401-jmrp" src="https://github.com/getsentry/sentry/assets/187460/e23b51a8-1c45-4d5a-b159-ed26c0211451">

If we want the semantics/spacing of the `<p>` then wrap it like `<p><pre>{message}</pre></p>`